packages/app/src/runner/aut-iframe.ts

```diff
@@ -93,7 +93,7 @@ export class AutIframe {
    * Otherwise, if top and the AUT match origins, the method returns true.
    * If the AUT origin is "about://blank", that means the src attribute has been stripped off the iframe and is adhering to same origin policy
    */
-  doesAUTMatchTopOriginPolicy = () => {
+  doesAUTMatchTopSuperOriginPolicy = () => {
```

Comment:
Should this be:
```suggestion
  doesAUTMatchTopSuperDomainOriginPolicy = () => {
```

Reply:
fixed in [a0de810](https://github.com/cypress-io/cypress/pull/23879/commits/a0de810559499cc4b93ab32dab3ae5bfb8d767b6)